.github/workflows/test-publish.yaml

```diff
@@ -31,4 +31,4 @@ jobs:
             cache-gradle-
 
       - name: Run the uploadSentryProguardMappings task
-        run: ./gradlew assembleRelease uploadSentryProguardMappingsReleaseRelease
+        run: ./gradlew assembleRelease uploadSentryProguardMappingsRelease
```

Comment:
Commenting here but this is related to the whole PR:

This is basically a breaking change for the API exposed to customers.
This means that any build logic (`build.gradle` or custom plugins) that rely on this plugin, will break.

I don't think this is a major concern, but it worth to point it out clearly in the release notes.

Reply:
the task is called automatically, we never expect people calling this directly, so I don't think its a breaking change

Reply:
> we never expect people calling this directly

😅 

I'm not entirely sure here, but if someone has a `./gradlew uploadSentryProguardMappingsReleaseRelease` in their CI build because they want to explicitly be sure that the upload happens -> Their CI step will break.

Reply:
it'd be an undocumented API, it's fine in this case, our docs say, set up the plugin and that's it, tasks are called automatically.
obviously, we can mention in the changelog